src/dotless.Test/Specs/RulesetsFixture.cs

```diff
@@ -135,5 +135,135 @@ public void IESlash9HackAllowedForNonEvaluatedValues()
             var expected = input;
             AssertLess(input, expected);
         }
+
```

Comment:
Your quite right, I should have made more cases to cover that, did not need it for the problem I had to solve but I imagine we will hit that to very soon now that we have solved the original problem.